src/cmd_all/src/playground.rs

```diff
@@ -286,9 +287,12 @@ pub async fn playground(opts: PlaygroundOpts) -> Result<()> {
             risedev_env,
         )?;
 
-        "./risedev psql"
+        "./risedev psql".to_string()
     } else {
-        "psql -h localhost -p 4566 -d dev -U root"
+        format!(
+            "psql -h localhost -p 4566 -d {} -U {}",
+            *DEFAULT_DATABASE_NAME, *DEFAULT_SUPER_USER
+        )
```

Comment:
This actually reminds me of the `PGxxx` environment variables..

https://www.postgresql.org/docs/current/libpq-envars.html